vuepress/docs/next/tutorials/devops/plugin-environment-variables.md

```diff
@@ -0,0 +1,68 @@
+---
+sidebarDepth: 2
+---
+
+# Use Plugin Environment Variables to Customize Microservices
+
+This tutorial describes how to use environment variables to customize plugin microservices behavior. Using environment variables as a configuration mechanism allows the same Linux image to run in development and production environments. This is especially useful when managing sensitive information via Kubernetes Secrets.
+
+Version 4 of the Entando plugin descriptor adds the ability to make environment variables available to a microservice via the plugin `descriptor.yaml` file. There are two options.
+1. Inject the variables using a key-value pair
+2. Inject a reference to an existing Kubernetes Secret 
+
+This tutorial will demonstrate both of these options. The Entando plugin will receive one environment value directly as plain text in the pod YAML and the other indirectly via a referenced Secret.
```

Comment:
the Entando Glossary (google sheet) seems geared towards the correct words rather than capitalization conventions... not sure if the note that Plugin -> plugin means that this term should be lowercase; it was capitalized when directly following "Entando" in the release notes doc so i adopted that in my rework. same with "pod" - i noticed it was capitalized in the quickstart so kept that up on that page...